File: 1008/1008.c
#include <stdio.h>

int main() {
    int n, m;
    float s;
    scanf("%d %d %f", &n, &m, &s);
    
    printf("NUMBER = %d\nSALARY = U$ %.2f\n", n, m *s);
    return 0;
}

File: 1. Beginner/1008/1008.ex
n = IO.gets("") |> String.trim |> String.to_integer
{m, _} = IO.gets("") |> String.trim |> Float.parse
{s, _} = IO.gets("") |> String.trim |> Float.parse

IO.puts "NUMBER = #{n}\nSALARY = U$ #{:erlang.float_to_binary(m * s, [decimals: 2])}"
